frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -161,19 +163,27 @@ export default class QueryBuilder extends Component {
     }
 
     componentWillUnmount() {
-        window.removeEventListener('resize', this.handleResize);
+        window.removeEventListener("resize", this.handleResize);
+        document.addEventListener("keydown", this.handleKeyDown);
     }
 
     // When the window is resized we need to re-render, mainly so that our visualization pane updates
     // Debounce the function to improve resizing performance.
-    handleResize(e) {
+    handleResize = (e) => {
         this.forceUpdateDebounced();
         let viz = ReactDOM.findDOMNode(this.refs.viz);
         if (viz) {
             viz.style.opacity = 0.2;
         }
     }
 
+    handleKeyDown = (e) => {
+        const ENTER_KEY = 13;
+        if (e.keyCode === ENTER_KEY && e.metaKey && this.props.isRunnable) {
```

Comment:
I thought Meta is supposed to be the same thing as Alt/Option, right? At least it is in Terminal.app and Emacs